lib/src/screens/dashboard/dashboard_page.dart

```diff
@@ -158,102 +167,106 @@ class _DashboardPageView extends BasePage {
 
     return SafeArea(
       minimum: EdgeInsets.only(bottom: 24),
-      child: Column(
-        mainAxisSize: MainAxisSize.max,
-        children: <Widget>[
-          Expanded(
-            child: Observer(
-              builder: (context) {
-                return PageView.builder(
-                  controller: controller,
-                  itemCount: pages.length,
-                  itemBuilder: (context, index) => pages[index],
-                );
-              },
-            ),
-          ),
-          Padding(
-            padding: EdgeInsets.only(bottom: 24, top: 10),
-            child: Observer(
-              builder: (context) {
-                return ExcludeSemantics(
-                  child: SmoothPageIndicator(
+      child: BottomSheetListener(
```

Comment:
you haven't added it in case of DesktopDashboardPage

Reply:
True, I'll add it in